### DISCUSSION TITLE
renovate does not preserve/respect tag comments for pinned SHA in `docker://` github action

### DISCUSSION THREAD

**Discussion by lucasrod16** [renovatebot/renovate#28323] [Request Help] [answered]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

renovate is stripping away the tag comment from a pinned docker image SHA in a `docker://` github action:

[chore(deps): update morphy/revive-action docker digest to 087d4e6 #2434 ](https://github.com/defenseunicorns/zarf/pull/2434/files)

I expect renovate to preserve the tag comment:

https://docs.renovatebot.com/modules/manager/github-actions/#additional-information

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Instead of this:

```
docker://morphy/revive-action@sha256:087d4e61077087755711ab7e9fae3cc899b7bb07ff8f6a30c3dfb240b1620ae8 # v2.5.7
```

Use this:

```
docker://morphy/revive-action:2.5.7@sha256:087d4e61077087755711ab7e9fae3cc899b7bb07ff8f6a30c3dfb240b1620ae8
```

**Reply by lucasrod16** [reply]
@rarkins thank you! This fixed the issue

https://github.com/defenseunicorns/zarf/pull/2436
https://github.com/defenseunicorns/zarf/pull/2434
